src/lib/tooltip/tooltip.ts

```diff
@@ -309,6 +310,10 @@ export class TooltipComponent {
       // If this was set to true immediately, then a body click that triggers show() would
       // trigger interaction and close the tooltip right after it was displayed.
       this._closeOnInteraction = false;
+
+      // Call change detection manually so tooltip will also work 
+      // if any parent component has set the ChangeDetectionStrategy to OnPush
+      this._ref.detectChanges();
```

Comment:
@jelbourn is there a way to find out if a parent component has set the `ChangeDetectionStrategy` to `OnPush` or not so we can skip this line?